### DISCUSSION TITLE
How do I get the Yolo badge ?

### DISCUSSION THREAD

**Discussion by perk-bevis** [community/community#58015] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body
How do I get the Yolo badge?
https://github.com/Schweinepriester/github-profile-achievements

**Comment by SuperStar0718** [comment]
OK thanks

**Comment by Gustavonn07** [comment, marked as answer]
You can see in this repository how to get all the achievements https://github.com/4xmen/Get-Github-Achievements

1. First you need to fork from a repository , click fork then click on Create fork button get a fork of a repository (person who will make pullrequest)

2. Add a file edit some texts or do some changes on forked repository (person who will make pullrequest)

3. Commit your added file or edited text in forked repository . (person who will make pullrequest)

4. Create a new branch for this commit and start a pull request (person who will make pullrequest)

5. Click , Compare and pull request button (person who will make pullrequest)

6. Add a Reviewer To you committed changes on forked repository (you)

7. Add Any comment you want in the box then click on Create pull request . When your Pull Request added - Merge it before the Reviewer and Click on Merge Pull Request button . (you)

8. Done , Now you can see that Yolo Achievement in your achievements list.

**Reply by AlyaniMamad** [reply]
You can see in this repository how to get all the achievements https://github.com/4xmen/Get-Github-Achievements

First, you need to fork from a repository, click fork then click on Create fork button to get a fork of a repository (the person who will make pull request)

Add a file edit some texts or make some changes on the forked repository (the person who will make pull-request)

Commit your added file or edit text in a forked repository. (person who will make pull request)

Create a new branch for this commit and start a pull request (person who will make pull request)

Click, Compare, and pull the request button (person who will make pull-request)

Add a Reviewer To your committed changes on the forked repository (you)

Add Any comment you want in the box then click on Create pull request. When your Pull Request is added, Merge it before the Reviewer and Click on the Merge Pull Request button. (you)

Done, Now you can see Yolo's Achievement in your achievements list.

**Comment by dreamcatcher1020** [comment]
Hello.
[](https://github.com/4xmen/Get-Github-Achievements/blob/main/assets/steps/README.yolo.md)
You can follow step by step as Gustavonn07 said.
In my experience, the most important section is to change the same file on two github accounts concurrently.

**Comment by Atharva404** [comment]
https://github.com/4xmen/Get-Github-Achievements

This has been a big help
